app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
It's an important piece of information. In fact, it needs to be searchable. Moving the info into active record would speed this up and allow search, right?